api/src/main/resources/Reference_Application_Diagnoses-13.xml

```diff
@@ -320,7 +320,7 @@
 <concept_description concept_description_id="9072" concept_id="129566" description="A developmental anomaly characterized by the presence of more than five fingers on the hand or more than five toes on the foot in primates. In non-primate vertebrates, it refers to the presence of more than the normal number of toes on the forelimb or hindlimb. (Dorland, 27th ed; Stedman, 25th ed)" locale="en" creator="1" date_created="2007-10-18 04:28:24.0" changed_by="1" uuid="9072FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"/>
 <concept_description concept_description_id="9467" concept_id="131113" description="Inflammation of the middle ear." locale="en" creator="1" date_created="2007-11-02 20:16:26.0" changed_by="1" uuid="9467FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"/>
 <concept_description concept_description_id="9468" concept_id="131113" description="Otite moyenne" locale="fr" creator="1" date_created="2007-09-06 08:51:25.0" changed_by="1" uuid="9468FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"/>
-<concept_description concept_description_id="9762" concept_id="132636" description="A syndrome produced by severe protein deficiency, characterized by retarded growth, changes in skin and hair pigment, edema, and pathologic changes in the liver, including fatty infiltration, necrosis, and fibrosis. The word is a local name in Gold Coast, Africa, meaning "displaced child". Although first reported from Africa, kwashiorkor is now known throughout the world, but mainly in the tropics and subtropics. It is considered to be related to marasmus. (From Dorland, 27th ed)" locale="en" creator="1" date_created="2007-10-18 04:28:24.0" changed_by="1" uuid="9762FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"/>
+<concept_description concept_description_id="9762" concept_id="132636" description="A syndrome produced by severe protein deficiency, characterized by retarded growth, changes in skin and hair pigment, edema, and pathologic changes in the liver, including fatty infiltration, necrosis, and fibrosis. The word is a local name in Gold Coast, Africa, meaning displaced child. Although first reported from Africa, kwashiorkor is now known throughout the world, but mainly in the tropics and subtropics. It is considered to be related to marasmus. (From Dorland, 27th ed)" locale="en" creator="1" date_created="2007-10-18 04:28:24.0" changed_by="1" uuid="9762FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"/>
```

Comment:
What is the purpose of this change?

Reply:
With this change i was making it a full  single string by removing double quotes on "display child" which was a quick fix

Reply:
What error was it giving?

Reply:
Element type "concept_description" must be followed by either attribute specifications, ">" or "/>" element unterminated thats was the error it was giving